Parking-Zone/Areas/Admin/Views/ParkingSlot/Index.cshtml

```diff
@@ -4,45 +4,172 @@
+                <option value="Standard">Standard</option>
+                <option value="Business">Business</option>
+            </select>
+        </div>
+        <div class="custom-checkbox">
+            <input class="form-check-input" type="checkbox" id="checkboxID" style="margin-bottom: 10px;" />
+            <label class="form-check-label" for="checkboxID">Display free slots</label>
+        </div>
+
+        <button class="btn btn-primary" type="submit" id="submitButton">Filter</button>
+    </div>
+
+    <div class="container" style="margin-left: 530px;">
+        <h1>Slots for @ViewData["parkingZoneName"]</h1>
+        <p>
+            <a asp-action="Create" asp-route-zoneId="@ViewData["parkingZoneId"]" class="btn btn-primary">Create New</a>
+        </p>
+    </div>
+</div>
+<br />
+<div id="result">
+    <table class="table table-striped">
+        <thead>
             <tr>
-                <td>
-                    @Html.DisplayFor(modelItem => item.Number)
-                </td>
-                <td>
-                    @Html.DisplayFor(modelItem => item.IsAvailableForBooking)
-                </td>
-                <td>
-                    @Html.DisplayFor(modelItem => item.Category)
-                </td>
-                <td>
-                    <a asp-action="Edit" asp-route-id="@item.Id">Edit</a> |
-                    <a asp-action="Details" asp-route-id="@item.Id">Details</a> |
-                    <a asp-action="Delete" asp-route-id="@item.Id">Delete</a>
-                </td>
+                <th>
+                    @Html.DisplayNameFor(model => model.Number)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.IsAvailableForBooking)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Category)
+                </th>
+                <th>
+                    Actions
+                </th>
             </tr>
-        }
-    </tbody>
-</table>
\ No newline at end of file
+        </thead>
+
+        <tbody id="tableBody">
+            @foreach (var item in Model)
+            {
+                <tr>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.Number)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.IsAvailableForBooking)
+                    </td>
+                    <td>
+                        @Html.DisplayFor(modelItem => item.Category)
+                    </td>
+                    <td>
+                        <a asp-action="Edit" asp-route-id="@item.Id" class="btn btn-warning">Edit</a>
+                        <a asp-action="Details" asp-route-id="@item.Id" class="btn btn-success">Details</a>
+                        <a asp-action="Delete" asp-route-id="@item.Id" class="btn btn-danger">Delete</a>
+                    </td>
+                </tr>
+            }
```

Comment:
these lines are not necessary. Use jquery to Populate table this code has written below